manifests/0000_90_ingress-operator_03_prometheusrules.yaml

```diff
@@ -53,3 +53,14 @@ spec:
           message: |
             The {{ $labels.namespace }}/{{ $labels.name }} ingresscontroller is
             unavailable: {{ $labels.reason }}.
+      # Recording rules related to route metrics for sending via telemetry
+      - expr: min(route_metrics_controller_routes_per_shard)
```

Comment:
Might be a dumb question, but looks like we get min, max, avg, quantile, but should also get back the instantaneous value of `route_metrics_controller_routes_per_shard` without any mathematical functions? This may have already been reviewed and decided on, in that case, I don't have a strong opinion, just curious.

Reply:
There's an issue with the metric budget to be sent via telemetry. `route_metrics_controller_routes_per_shard` has a label called `name` and it doesn't have any limit on the values it can take. It is completely dependent on the cluster and the admin who creates the shards. This makes the metric unbounded and thus not fit for sending via telemetry. After a lot of deliberation it was decided that the min, max, avg and median values would be within the budget and also would be able to give an overall understanding of the shard configuration of a cluster. There were previous discussions which you can find in here: https://coreos.slack.com/archives/C03SXF61V50/p1660911654536479

Reply:
Information about sending metrics via telemetry can be found in this document: https://docs.google.com/document/u/3/d/1a6n5iBGM2QaIQRg9Lw4-Npj6QY9--Hpx3XYut-BrUSY/edit